File: cv-banken-vue/src/models/requests/UpdateProfileRequest.js
export const updateProfileModel = request => {
    return {
        firstName: request.firstName,
        lastName: request.lastName,
        programmeId: request.programmeId,
        email: request.email,
        oldPassword: request.oldPassword,
        password: request.password,
        private: request.private,
        searching: request.searching,
        description: request.description,
    }

}

File: CVBanken.Data/Models/UpdateProfileRequest.cs
using Microsoft.AspNetCore.Http;

namespace CVBanken.Data.Models
{
    public class UpdateProfileRequest
    {
        public UpdateUser User { get; set; }
        public bool Private { get; set; }
        public bool Searching { get; set; }
        public string Description { get; set; }
        public virtual IFormFile ProfilePicture { get; set; }
    }
    public class UpdateUser
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int ProgrammeId { get; set; }
        public string Email { get; set; }
        public string OldPassword { get; set; }
        public string Password { get; set; }
    }
    
}